client/app/states/designer/blueprints/editor/editor.html

```diff
@@ -1,15 +1,14 @@
-<span ng-controller="BlueprintEditorController" ng-class="{'wait-cursor': loading}">
+<span ng-controller="BlueprintEditorController as evm" ng-class="{'wait-cursor': loading}">
```

Comment:
ooh ooh editor view model and canvas view model!  I get it <3

one of these days we'll have to componentize these puppies... one of these days...